src/client/helpers/entity.js

```diff
@@ -210,3 +210,23 @@ export function genEntityIconHTMLElement(entityType, size = '', margin = true) {
 export function getSortNameOfDefaultAlias(entity) {
 	return entity.defaultAlias.sortName;
 }
+
+export function getISBNOfEdition(entity) {
+	if (entity.identifierSetId) {
+		const {identifiers} = entity.identifierSet;
+		let isbn = null;
+		isbn = identifiers.find(identifier => identifier.type.label === 'ISBN-13');
```

Comment:
This can be refactored as below to avoid running .find twice:

`identifiers.find(identifier => identifier.type.label === 'ISBN-13' || identifier.type.label === 'ISBN-10' );`

Reply:
Acutally, I think you can `return` that `identifiers.find` directly.
It will return undefined if it doesn't find any, and considering how you're using it in edition-table.js that will work fine.